src/ui/GenioWindow.cpp

```diff
@@ -1522,15 +1559,13 @@ GenioWindow::_FileSave(int32 index)
 	}
 */
 	// Stop monitoring if needed
-	if (editor->StopMonitoring() != B_OK)
-		LogErrorF("Error in StopMonitoring node (%s)", editor->Name().String());
+	editor->StopMonitoring();
 
 	ssize_t written = editor->SaveToFile();
 	ssize_t length = editor->SendMessage(SCI_GETLENGTH, 0, 0);
 
 	// Restart monitoring
-	if (editor->StartMonitoring() != B_OK)
```

Comment:
Here we are removing a previous change to log an error in case of start monitor failure